app/controllers/image_uploads_controller.rb

```diff
@@ -30,4 +33,20 @@ def create
       format.json { render json: { link: link }, status: 200 }
     end
   end
+
+  def limit_uploads
+    count = Rails.cache.read("#{current_user.id}_image_upload")
+
+    if count.nil?
```

Comment:
Is there a better way to write this? I tried shortening it a few different ways but they all raised some sort of code climate issue.

Reply:
You could try:
```ruby
count = Rails.cache.read("#{current_user.id}_image_upload").to_i
count += 1
```
`nil` gets converted to `0` like this so you can avoid the `if` statement 👍